backend/models/item.js

```diff
@@ -0,0 +1,17 @@
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
+
+var BatchSchema = new Schema({
```

Comment:
Can both of these schemas also be changed to const, since we wont be changing any of these variables later in the code?